packages/fireprox/PKGBUILD

```
@@ -0,0 +1,48 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=fireprox
+pkgver=51.a6e1300
+pkgrel=1
+pkgdesc='AWS API Gateway management tool for creating on the fly HTTP pass-through proxies for unique IP rotation.'
+arch=('any')
+groups=('blackarch' 'blackarch-webapp')
+url='https://github.com/ustayready/fireprox'
+license=('GPL3')
+depends=('python' 'python-virtualenv')
+makedepends=('git' 'python-setuptools' 'python-pip')
```

```suggestion
makedepends=('git' 'python-pip')
```

there is not package, no setup.py